### ISSUE TITLE
🚌  Bus factor

### ISSUE THREAD

**Issue by jefftriplett** [djangopackages/djangopackages#1029]
I haven't been comfortable with our bus factor for a while, and the recent add/edit form break is a good example of the issue. I'm happy to keep the project management loose, but right now I am the only person who can deploy code. Ideally, this should be controlled within GitHub (maybe releases instead of on merging into main). 

We also have admin/maintainers whom I have never met or heard from despite my creating a few group discussions in the past which is probably a sign that we need to clean some of that up too. 

Instead of creating a discussion, this now feels more of an issue that I'd like to start resolving over the next month or so.

**Comment by ryancheley** [comment]
@jefftriplett perhaps this can be something to work on as part of the contribution sprints at DCUS next month?

**Comment by jefftriplett** [comment]
@ryancheley I welcome this. I'll work on that only deploys from my laptop issue because I had to VNC home earlier just to deploy it. Not ideal at all.

**Comment by jefftriplett** [comment]
One workaround here is to update our GitHub Actions to deploy on the release event and update the project. I have everything on Tailscale, so it needs their action to setup tailscale and then `tailscale ssh ...` possibly to connect to restart services.

**Comment by saadmk11** [comment]
> One workaround here is to update our GitHub Actions to deploy on the release event and update the project. I have everything on Tailscale, so it needs their action to setup tailscale and then `tailscale ssh ...` possibly to connect to restart services.

Good idea. Tailscale has a good blog post about it https://tailscale.com/blog/2021-05-github-actions-and-tailscale/

**Comment by jefftriplett** [comment]
I'm going to close this due to being a better place thanks to you two. @ryancheley @saadmk11
